src/evergreen/task.py

```diff
@@ -31,6 +32,29 @@ def __init__(self, json, api):
         super(Artifact, self).__init__(json, api)
 
 
+class StatusScore(IntEnum):
+    SUCCESS = 1
+    FAILURE = 2
+    FAILURE_TIMEOUT = 3
+    FAILURE_SYSTEM = 4
+    UNDISPATCHED = 5
+
+    @classmethod
+    def get_task_status_score(self, task):
+        if task.is_success():
+            return StatusScore.SUCCESS
+        if not task.is_success() and not task.is_system_failure() and not task.is_timeout() and \
```

Comment:
Since are you are returning in each of these `if` statements, you can probably simplify the checks. Also, if you reorder the checks, I think you can simplify even more:
```
if task.is_success():
    return StatusScore.SUCCESS
if task.is_undispatched():
    return StatusScore.UNDISPATCHED
if task.is_timeout():
    return StatusScore.FAILURE_TIMEOUT
if task.is_system_failure():
    return StatusScore.FAILURE_SYSTEM
return StatusScore.FAILURE
```